api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -571,10 +575,12 @@ public CohortDefinition testedInLast12Months() {
      * Tested more than twice in the last 12 Months
      * @return CohortDefinition
      */    
-	public CohortDefinition testedMoreThanTwiceInLast12Months() {
```

Comment:
@solemabrothers Why delete the old definition which is used in previous reports? They are now broken. Just create a new definition and leave the old one untouched